next/next.config.js

```diff
@@ -8,7 +8,7 @@ const nextConfig = {
   i18n,
   reactStrictMode: true,
   images: {
-    domains: ['localhost', 'cdn-api.bratislava.sk', 'goout.net', 'api.tootoot.co'],
+    domains: ['localhost', 'cdn-api.bratislava.sk', 'api.tootoot.co'],
```

Comment:
Please try if this works:
```
    remotePatterns: [
      {
        protocol: 'http',
        hostname: 'localhost'
      },
      {
        protocol: 'https',
        hostname: 'cdn-api.bratislava.sk'
      },
      {
        protocol: 'https',
        hostname: 'api.tootoot.co'
	  pathname: '/api/event/*/images/*/*/*/AUTO',
      },
    ],
    ```
    
    We should not use domains: https://nextjs.org/docs/app/api-reference/components/image#domains

Reply:
Nice, it works fine :)